.env.development

```diff
@@ -6,3 +6,4 @@ DB_NAME=dev-directory
 DISCORD_GUILD_ID=735923219315425401
 DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
 NEXT_PUBLIC_DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
+DATABASE_URL = postgres://postgres:password@localhost:5432/dev-directory
```

Comment:
oop - i mean, in this file we want there to be _no_ space (see examples on previous line) 🙃